src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -199,7 +205,9 @@ export class FileExplorerComponent implements OnInit {
 
   getDelimiter(selectedFolder) {
     const path = this.breadcrumbPath.substring(this.breadcrumbPath.indexOf(this.DELIMITER), this.breadcrumbPath.length);
-    return path.substring(1, path.lastIndexOf(selectedFolder)) + selectedFolder;
+    const indexSelectedFolder = path.lastIndexOf('/' + selectedFolder + '/') !== -1 ? path.lastIndexOf('/' + selectedFolder + '/') + 1
```

Comment:
This would be clearer if `path.lastIndexOf('/' + selectedFolder + '/')` was stored in a variable, so it isn't duplicated in the code.